src/domain/enrolment/__tests__/CreateEnrolmentPage.test.tsx

```diff
@@ -398,66 +398,74 @@ test('render and focuses error notification correctly', async () => {
   expect(screen.getByRole('textbox', { name: 'Nimi' })).toHaveFocus();
 });
 
-test('amount of adults is limited to 1 when there is 1 seat left after children', async () => {
-  render(<CreateEnrolmentPage />, {
-    mocks: mock3,
-    query: { eventId: eventId, occurrences: occurrenceIds },
-  });
-  await waitFor(() => {
-    expect(screen.getByLabelText(/lapsia/i)).toBeInTheDocument();
-  });
-  userEvent.type(screen.getByLabelText(/lapsia/i), '19');
-  userEvent.type(screen.getByLabelText(/aikuisia/i), '2');
-  userEvent.tab();
-  await waitFor(() => {
-    expect(
-      screen.getByText(
-        /Arvon tulee olla enintään 1 yhdessä lasten lukumäärän kanssa/i
-      )
-    ).toBeInTheDocument();
+describe('max group size validation of the Children and Adults -fields', () => {
+  const createEnrolmentForm = async (
+    childrenCount: string,
+    adultsCount: string
+  ) => {
+    render(<CreateEnrolmentPage />, {
+      mocks: mock3,
+      query: { eventId: eventId, occurrences: occurrenceIds },
+    });
+    await waitFor(() => {
+      expect(screen.getByLabelText(/lapsia/i)).toBeInTheDocument();
+    });
+    userEvent.type(screen.getByLabelText(/lapsia/i), childrenCount);
+    userEvent.type(screen.getByLabelText(/aikuisia/i), adultsCount);
+    userEvent.tab();
+  };
+
+  test('both of the fields are greater than the max group size', async () => {
+    createEnrolmentForm('21', '22');
+    await waitFor(() => {
+      expect(
+        screen.getAllByText(
+          /Lasten ja aikuisten yhteislukumäärän tulee olla enintään 20/i
+        )
+      ).toHaveLength(2);
```

Comment:
I was thinking if also these error messages could be passed as parameters but this one of course causes some problems.